src/js/game/LevelMVC/LevelView.js

```diff
@@ -128,9 +128,19 @@ module.exports = class LevelView {
       "railsPoweredVertical": ["blocks", "Rails_PoweredVertical", -13, 0],
       "railsRedstoneTorch": ["blocks", "Rails_RedstoneTorch", -12, 9],
 
-      "redstone_dust": ["blocks", "Redstone_Dust_Vertical", -13, 0],
+      "redstoneWire": ["blocks", "redstone_dust_dot_off", -13, 0],
+      "redstoneWireHorizontal": ["blocks", "redstone_dust_line_h_off", -13, 0],
+      "redstoneWireVertical": ["blocks", "redstone_dust_line_v_off", -13, 0],
+      "redstoneWireUpRight": ["blocks", "redstone_dust_corner_BottomLeft_off", -13, 0],
+      "redstoneWireUpLeft": ["blocks", "redstone_dust_corner_BottomRight_off", -13, 0],
+      "redstoneWireDownRight": ["blocks", "redstone_dust_corner_TopLeft_off", -13, 0],
+      "redstoneWireDownLeft": ["blocks", "redstone_dust_corner_TopRight_off", -13, 0],
+      "redstoneWireTUp": ["blocks", "redstone_dust_cross_up_off", -13, 0],
+      "redstoneWireTDown": ["blocks", "redstone_dust_cross_down_off", -13, 0],
+      "redstoneWireTLeft": ["blocks", "redstone_dust_cross_left_off", -13, 0],
+      "redstoneWireTRight": ["blocks", "redstone_dust_cross_right_off", -13, 0],
+      "redstoneWireCross": ["blocks", "redstone_dust_cross_off", -13, 0],
```

Comment:
Tiny nit: with the minecart tracks I've found it a lot easier to refer to them by north/south/east/west instead of up/down/right/left, since "up" as in "above a block" is different from "up" as in "north of a block".  Might be clearer to follow that pattern here?

Reply:
These I just took the JSON names as they were given to me by Joel, since that what he named his files. What do you mean about up vs above a block? I could understand if we were 3D, but since we're just a 2D array of blocks, I felt like "up" or "above" was still pretty clear? I can go through and change it if you like.

Even if we don't I can use north/south from now on if you want? Just let me know.

Reply:
I think north/south/east/west is clearer for the reason I mentioned above, but certainly not blocking for this PR.